src/redux/reducers/usersReducer.js

```diff
@@ -0,0 +1,81 @@
+import { FOLLOW, UNFOLLOW, SET_USERS } from '../actions/actionTypes';
+import { AVATAR_DEFAULT } from '../../utils/url-utils';
+
+const initialState = {
+  users: [
```

Comment:
Consider moving this to a separate json file.

Reply:
in the very near future I will receive this data using the request